projectq/ops/_command.py

```diff
@@ -152,6 +151,24 @@ def is_identity(self):
             True if the gate is equivalent to an Identity gate, False otherwise
         """
         return projectq.ops.is_identity(self.gate)
+    
+    def is_commutable(self, other):
```

Comment:
Done. is_commutable now returns a value such as: Commutable.COMMUTABLE.value. I chose to keep it returning an integer (the value of COMMUTABLE) rather than an enum, because then in my test files I can keep the nice simple readability of ‘assert gate1.is_commutable(gate2)’ and ‘assert not gate2.is_commutable(gate3)’. For the third option (commutable circuit) you use: ‘assert gate3.is_commutable(gate4) == Commutable.MAYBE_COMMUTABLE

Reply:
Instead of an enum class deriving from `Enum`, why don't you consider deriving from `IntEnum`?
The latter gives you conversion to `int` without needing to access the `value` attribute.

Reply:
Done. As will be seen in the next commit.